products/docker-engine.md

```diff
@@ -63,7 +63,7 @@ releases:
 
 -   releaseCycle: "23.0"
     releaseDate: 2023-02-02
-    eol: false
+    eol: 2025-02-01 # estimated, see https://github.com/moby/moby/blob/master/project/BRANCHES-AND-TAGS.md
```

Comment:
Mirantis will be maintaining the 23.0 branch until 2025-05-19.  https://docs.mirantis.com/mcr/23.0/compat-matrix/maintenance-lifecycle.html#mirantis-container-runtime-mcr

Reply:
Could you update https://github.com/moby/moby/blob/master/project/BRANCHES-AND-TAGS.md